source/index.rst

```diff
@@ -1,53 +1,35 @@
 Welcome to NanoVer's documentation!
 ===================================
 
-* `Download the VR client <https://github.com/IRL2/nanover-imd/releases/download/nightly/StandaloneWindows64.zip>`_
-* :ref:`Install the software <Installation>`
-* :ref:`Read tutorials <Tutorials>`
+NanoVer is a free, open-source and flexible software that can be used as:
```

Comment:
Do we want to state that it is under the **MIT license** and reference it?

Reply:
This was moved to the Highlights section, but I'll put it back here as well.